src/schema/v2/artist/insights.ts

```diff
@@ -36,6 +37,13 @@ export const ArtistInsight = new GraphQLObjectType<any, ResolverContext>({
     },
     description: {
       type: GraphQLString,
+      args: {
+        ...markdown().args,
+      },
+      resolve: ({ description }, { format }) => {
+        if (!format) return formatMarkdownValue(description, "plain").trim()
```

Comment:
@dzucconi we are avoid breaking changes here

Reply:
You should be able to just use the normal markdown field in it's entirety, no?

Reply:
it doesn't have the defaultValue